### PR TITLE
Add all properties of remediations

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
### Description
All the properties (including 6 optional properties) of remediations are processed. Previously we added only the `details` property.
So, the vulnerability['remediations'] returns all the information associated with it.

### CODE REVIEW

**Review by individual-it — APPROVED** [review]

**Comment by Salipa-Gurung** [comment]
Sorry, let's not merge this pr until we have our pr ready to be merged in `RSSCollector` repo for remediations.
Until then keeping this pr in draft.

**Review by S-Panta — APPROVED** [review]

**Comment by Salipa-Gurung** [comment]
we will be working on remediations in pr: https://github.com/JankariTech/RSSCollector/pull/49 or in another follow up pr.

**Pull request merged**
